### ISSUE TITLE
aria-errormessage wrongfully throws "[aria-*] attributes do not have valid values" 

### ISSUE THREAD

**Issue by cristurm** [GoogleChrome/lighthouse#12379]
#### Provide the steps to reproduce
**Code example**
```
<input type="radio" name="example" id="example-a" aria-invalid="true" aria-errormessage="err-msg" value="a">
<label for="example-a">A</label>

<input type="radio" name="example" id="example-b" aria-invalid="true" aria-errormessage="err-msg" value="b">
<label for="example-b">B</label>

<span id="err-msg">This field is required</span>
```

Run Chrome Dev Tools - Lighthouse - Accessibility report on any form with `aria-errormessage` attribute.

#### What is the current behavior?
* It fails with this error: `[aria-*] attributes do not have valid values`
* If I change the value to `aria-errormessage="#err-msg"` (add the # - ID selector - to it) it passes. 

#### What is the expected behavior?
* It shouldn't fail without the ID selector (#) (In fact, I believe it should fail with it)
* The ARIA documentation states the `aria-errormessage` value should be an ID string, so the plain ID (without #) should suffice. 
* https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage

#### Environment Information
* Affected Channels: DevTools
* Lighthouse version: Lighthouse 7.2.0
* Chrome version: 90.0.4430.72 (Official Build) (64-bit) (cohort: 90_72_win)
* Node.js version: N/A
* Operating System: Windows 10

**Related issues**
Unknown

**Comment by patrickhulce** [comment]
Thanks for filing @cristurm! This is intentional behavior from `axe-core`, but the way Lighthouse is surfacing the information is missing critical pieces.

When `aria-errormessage` is set to a valid ID (`err-msg` in your example), axe checks for a valid announcement of the error message (see [axe code](https://github.com/dequelabs/axe-core/blob/b27bab3954f006e1257f7d70bd467991a2d9330e/lib/checks/aria/aria-errormessage-evaluate.js#L48-L54)), which your example lacks. axe returns this debug information in the form of `aria-errormessage value err-msg must use a technique to announce the message (e.g., aria-live, aria-describedby, role=alert, etc.)`

When `aria-errormessage` is set to a non-matching value like `#err-msg` or `definitely-madeup-and-does-not-exist`, axe flags the issue for manual review instead, which Lighthouse ignores because we only surface *definite* a11y issues in the Lighthouse report.

In short there are two issues at play here:
1. Lighthouse doesn't surface the important debug message that axe provides to understand why this element is being flagged. (Needs Lighthouse discussion on how to resolve)
2. Lighthouse is missing the invalid ID issue completely. (Bringing up to the axe team)

**Comment by connorjclark** [comment]
ref #12282

**Comment by patrickhulce** [comment]
So the invalid ID is not an issue because `aria-errormessage` can be applied even when there is no error yet (see https://github.com/dequelabs/axe-core/issues/2460#issuecomment-825884546). Seems like part 1) is the only thing left to resolve, which is tracked by #12282
